packages/edit-site/src/components/header/index.js

```diff
@@ -79,64 +84,107 @@ export default function Header( {
 	const displayBlockToolbar =
 		! isLargeViewport || deviceType !== 'Desktop' || hasFixedToolbar;
 
+	// Track mouse location.
+	const [ mouseLoc, setMouseLoc ] = useState( { x: 0, y: 0 } );
+	useEffect( () => {
+		const setMouseLocFromEvent = ( event ) =>
+			setMouseLoc( { x: event.clientX, y: event.clientY } );
+
+		window.addEventListener( 'mousemove', setMouseLocFromEvent );
```

Comment:
`non-blocking`:

I wonder if we could add an event listener to the editor itself instead of the window. It might be worth exploring if performance becomes a big concern. 🤔

Reply:
theoretically, yeah!  we should be able to and that would be more ideal IMO.  we should definitely try to do that if we go forward with this approach.